packages/auth0/test/auth0.test.ts

```diff
@@ -39,9 +39,11 @@ describe('Auth0 simulator', () => {
                   res.status(200).send('ok');
                 })
               },
-
             },
-            scenarios: { person }
+            scenarios: { person },
+            *effects(slice, faker) {
+              yield createUserFromPerson(slice, faker);
+            },
```

Comment:
Since there is only one effect, you can delegate directly to it:

```js
effects: (slice, faker) => createUserFromPerson(slice, faker)
```
or even

```js
effects: createuserFromPerson
```